clang/lib/Driver/Multilib.cpp

```diff
@@ -152,6 +180,7 @@ template <> struct llvm::yaml::MappingTraits<MultilibSerialization> {
   static void mapping(llvm::yaml::IO &io, MultilibSerialization &V) {
     io.mapRequired("Dir", V.Dir);
     io.mapRequired("Flags", V.Flags);
+    io.mapOptional("ExclusiveGroup", V.ExclusiveGroup);
```

Comment:
Could we just call it a `Group` for simplicity?

Reply:
I'll rename it if you like, but I worry that that might be ambiguous, or at least unclear. Within the general context of linking and libraries, "group" need not mean a _mutually exclusive_ group; it could mean a grouping of libraries for other purposes too, like a mutually _dependent_ group (you must select all of these or none).

Reply:
`ExclusiveGroup` looks good to me and I agree that `Group` could be confusing.

Reply:
Thanks, @MaskRay. I've left it as `ExclusiveGroup` for now.